qt/res/OrganicMaps.desktop

```diff
@@ -7,6 +7,6 @@ Comment[ru]=Подробная оффлайновая карта мира
 Icon=organicmaps
 TryExec=OMaps
 Exec=OMaps
-Categories=Maps;Education;Geography;Geoscience;Qt;
+Categories=Maps;Utility;Geography;Geoscience;Qt;
```

Comment:
As mentioned earlier https://github.com/flathub/app.organicmaps.desktop/issues/32#issuecomment-1546734605,
I see this change contrary to the trends, and to what the average map user would expect.

Reply:
More discoverability by users with the goal of attracting active contributors with the goal of building a more usable Linux version with the goal of making Linux users happy is preferred.

Reply:
>  Organic Maps fulfils a very different role in my life, as I use it for commuting to work and home and to travel

If you do all that on Linux, then big kudos for beta (alpha) testing. In that sense this change indeed better represents the ideal state. I just hope that we don't disappoint too many future potential users, who rely on us as a literal `Utility`, too early, before some major shortcomings are ironed out.